packages/contracts-bedrock/contracts/universal/CrossDomainMessenger.sol

```diff
@@ -28,6 +23,50 @@ contract CrossDomainMessengerLegacySpacer {
     address private spacer_0_0_20;
 }
 
+/**
+ * @custom:legacy
+ * @title CrossDomainMessengerLegacySpacer1
+ * @notice Contract only exists to add a spacer to the CrossDomainMessenger where the
+ *         PausableUpgradable and OwnableUpgradeable variables used to exist. Must be
+ *         the third contract in the inheritance tree of the CrossDomainMessenger.
+ */
+contract CrossDomainMessengerLegacySpacer1 {
+    /**
+     * @custom:legacy
+     * @custom:spacer __gap
+     * @notice Spacer for backwards compatibility.
```

Comment:
This first gap spacer was pretty confusing to me, apparently it comes from that terrible `ContextUpgradeable` that OwnableUpgradeable` inherits, but it's odd that it comes before Ownable. Could we just add a short comment here to clarify that?